lib/cext/preprocess.rb

```diff
@@ -228,6 +248,40 @@ def read_write_field(struct_var_name, field_name, leaking)
         replacement: MYSQL2_FREE_STATEMENT
       }
     ]
+  },
+  'byebug.c' => {
+    gem: 'byebug',
+    patches: [
+      {
+        match: /\b(static VALUE .*?) = (?:Qfalse|Qnil);/,
```

Comment:
`\w+` would be more precise and avoid potentially matching functions or so.

Reply:
Fixed.